packages/wrangler/src/hyperdrive/create.ts

```diff
@@ -37,13 +37,13 @@ export async function handler(
 
 	const url = new URL(args.connectionString);
 
+	if (url.protocol == "postgresql:" || url.protocol == "postgres:") {
```

Comment:
Can you also check if `port === ''` which still let's a user explicitly set a port – this seems to be forcibly overriding the port

So:
```suggestion
	if ((url.protocol == "postgresql:" || url.protocol == "postgres:") && port === "") {
```

Reply:
And add a test case which confirms explicitly setting a port to something other than 5432 still takes effect

Reply:
Ahh definitely, thanks for catching that.

Reply:
No problem! Can you add the test too?